2-mandatory/2-retrying-random-numbers.js

```diff
@@ -10,10 +10,17 @@ function generateRandomNumber() {
 }
 
 function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+    let number;
+    do {
+       number = generateRandomNumber();
+    }
+    while (number < 50)
```

Comment:
For this function, we want to return when the number is greater than 50 only. Is it possible for 50 to be returned and if so, could you change this line to only return greater than 50 and not 50?